docs/components/configurationPanel.md

```diff
@@ -219,6 +227,8 @@ unchanged, then the buttons will be disabled. Otherwise, the buttons are enabled
 
 - Returns: Nothing.
 
-Stores all the unsaved changes in Chrome's storage when the value of any configuration option is changed. These values
-are saved separately without applying the gamepad configuration and are used to prevent loss of unsaved configuration
-due to the panel getting closed accidentally. These values are restored when the panel is reopened.
+Stores all the unsaved changes in Chrome's storage (using
```

Comment:
You can leave out "in Chrome's storage" and move the other part out of the parentheses.